syntax = "proto3";
message User{
    string uid = 1;
    string name = 2;
    string address = 3;
    string email = 4;
    string psswd_hash =5;
    int32 project_count = 6;
}

message UserPortal{
    repeated User users = 1;
}